selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/administration/AdministrationLocator.java

```diff
@@ -652,9 +653,9 @@
   public static final By              ELEMENT_CHANGE_LANGUAGE_POPUP_TITLE                          =
                                                                           By.xpath(".//*[@id='UIMaskWorkspace']//h5");
 
-  public static final String          ELEMENT_CHANGELANGUAGE_LANGUAGE                              = "//*[text()='${language}']";
+  public static final SelenideElement         ELEMENT_CHANGELANGUAGE_LANGUAGE                              =$ (byClassName("uiChangeLanguageForm ")).find(byText("French"));
 
-  public static final String          ELEMENT_AVATAR_CHANGELANGUAGE_APPLY                          = "//*[text()='${text}']";
+  public static final SelenideElement       ELEMENT_AVATAR_CHANGELANGUAGE_APPLY                          =$(byText("Apply")) ;
```

Comment:
Don't modify an existed locator, it may impact other tests. if you can't reuse it, add a new one.